app/editor/src/features/content/form/ContentLabelsForm.tsx

```diff
@@ -0,0 +1,46 @@
+import { FormikSelect, FormikText } from 'components/formik';
+import { useFormikContext } from 'formik';
+import { FaTrash } from 'react-icons/fa';
+import { Button, ButtonVariant, Col, Row } from 'tno-core';
+
+import { labelOptions } from './constants';
+import { IContentForm } from './interfaces';
+import * as styled from './styled';
+
+/**
+ * Provides a way to view/edit content labels.
+ * @returns the ContentLabelsForm
+ */
+export const ContentLabelsForm: React.FC = () => {
+  const { values, setFieldValue } = useFormikContext<IContentForm>();
+
+  const handleDelete = (id: number, key: string, value: string) => {
```

Comment:
Very basic delete.  Didn't add prompt because this is just a simple way to demonstrate labels and NLP.  Also the delete will only occur if the user saves the changes.